server/datastore/mysql/mdm_test.go

```diff
@@ -618,9 +618,9 @@ func testBulkSetPendingMDMHostProfiles(t *testing.T, ds *Datastore) {
 	}
 	err = ds.BatchSetMDMProfiles(ctx, nil, macGlobalProfiles, winGlobalProfiles)
 	require.NoError(t, err)
-	//macGlobalProfiles, err = ds.ListMDMAppleConfigProfiles(ctx, nil)
-	//require.NoError(t, err)
-	//require.Len(t, macGlobalProfiles, 3)
+	// macGlobalProfiles, err = ds.ListMDMAppleConfigProfiles(ctx, nil)
+	// require.NoError(t, err)
+	// require.Len(t, macGlobalProfiles, 3)
```

Comment:
Should that be removed?

Reply:
This change was from my auto-formatter. I'm not sure why the lines were commented out in the first place. I've uncommented them and the test works.